src/admin-views/editor/panel/list.php

```diff
@@ -5,7 +5,7 @@
 
 /** @var Tribe__Tickets__Tickets_Handler $tickets_handler */
 $tickets_handler = tribe( 'tickets.handler' );
-$total_tickets = $tickets_handler->get_total_event_capacity( $post_id );
+$total_tickets = tribe_get_event_capacity( $post_id );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 3 spaces but found 1 space

<sub>hash: a4f54cf0e79892bebcddcb77be31492a</sub>